test/specs/reference/tests/02.reference_sort.js

```diff
@@ -0,0 +1,137 @@
+exports.execute = function (options) {
+
+    describe("For determining reference objects and it's child objects,", function () {
+        var catalog_id = process.env.DEFAULT_CATALOG,
+            schemaName = "reference_schema",
+            tableName = "sorted_table";
+
+        var multipleEntityUri = options.url + "/catalog/" + catalog_id + "/entity/" + schemaName + ":"
+            + tableName;
+
+
+        // Test Cases:
+        describe('Reference.sort() method, ', function() {
+            var reference, reference2, page;
+            var limit = 10;
+
+            it('resolve should return a Reference object that is defined.', function(done) {
```

Comment:
This test case should not be here. If you want to access the reference object, just use `beforeAll`.
```javascript
beforeAll(function(done) {
  options.ermRest.resolve(singleEnitityUri, {cid: "test"}).then(function(response) {
    reference = response.contextualize.record;
    done();
  }, function(err) {
    console.dir(err);
    done.fail();
  })
});
```